js/style/style.js

```diff
@@ -67,12 +62,12 @@ function Style(stylesheet, animationLoop, options) {
 
         if (stylesheet.sprite) {
             this.sprite = new ImageSprite(stylesheet.sprite);
-            this.sprite.on('load', this.fire.bind(this, 'change'));
```

Comment:
Is there some other mechanism now for triggering a repaint when the sprite finishes loading?